src/Autowhatever.js

```diff
@@ -281,7 +283,7 @@ export default class Autowhatever extends Component {
 
     return (
       <div {...theme(`react-autowhatever-${id}-container`, 'container', isOpen && 'containerOpen')}>
-        <input {...inputProps} />
+        {this.props.renderInput(inputProps)}
```

Comment:
Actually yes, i agree that this is probably the better way to do it. Do you want to merge the original pull request or do you want me to redo it with the prop named `inputElement` and adding a demo?
I suppose when propagating that change to react-autosuggest we can add some documentation there regarding that element being expected to expose the full API of the regular 'input' element, otherwise accessibility might break.